Documentation/guides/LayoutCodeBehind.md

```diff
@@ -104,19 +104,37 @@ then following type will be generated:
 ```csharp
 // Generated code
 namespace Binding {
-  sealed class Main : global::Xamarin.Android.Design.LayoutBinding
-  {
-    public Main (global::Xamarin.Android.Design.ILayoutBindingClient client);
-
-    public override int ResourceLayoutID => Resource.Layout.Main;
-   
-    public Button                          myButton                {get;}
-    public CommonSampleLibrary.LogFragment log_fragment            {get;}
-    public global::Android.App.Fragment    secondary_log_fragment  {get;}
+  sealed class Main : global::Xamarin.Android.Design.LayoutBinding {
+    public override int ResourceLayoutID => global::Xamarin.Android.Tests.CodeBehindFew.Resource.Layout.Main;
+
+    [global::Android.Runtime.PreserveAttribute (Conditional=true)]
+    public Main (
+      global::Android.App.Activity client,
+      global::Xamarin.Android.Design.OnLayoutItemNotFoundHandler itemNotFoundHandler = null)
+        : base (client, itemNotFoundHandler) {}
+
+    [global::Android.Runtime.PreserveAttribute (Conditional=true)]
+    public Main (
+      global::Android.Views.View client,
+      global::Xamarin.Android.Design.OnLayoutItemNotFoundHandler itemNotFoundHandler = null)
+        : base (client, itemNotFoundHandler) {}
+
+    Button __myButton;
+    public Button myButton => FindView (global::Xamarin.Android.Tests.CodeBehindFew.Resource.Id.myButton, ref __myButton);
+
+    CommonSampleLibrary.LogFragment __fragmentWithExplicitManagedType;
+    public CommonSampleLibrary.LogFragment log_fragment => FindFragment (global::Xamarin.Android.Tests.CodeBehindFew.Resource.Id.fragmentWithExplicitManagedType, __fragmentWithExplicitManagedType, ref __fragmentWithExplicitManagedType);
```

Comment:
I believe this property should now be `fragmentWithExplicitManagedType`, not `log_fragment`.